src/shared/components/query/QueryStore.ts

```diff
@@ -87,6 +88,9 @@ export class QueryStore
 		labelMobxPromises(this);
 		if (urlWithInitialParams)
 			this.setParamsFromUrl(urlWithInitialParams);
+

```

Comment:
are we sure it makes sense to override params from URL with params from query session?  i'm not suggesting otherwise i just want to make sure we thought about it